### DISCUSSION TITLE
importing a regular json file

### DISCUSSION THREAD

**Discussion by nrdxp** [cuelang/cue#980] [Q&A]
I'm trying use git revisions in a cue file that are specified in an automatically generated json lock file. Ideally, I'd like to import the json file and reference the revisions from there. How could I accomplish this?

**Comment by mpvl** [comment]
Aside from using `cue cmd`, there is currently no way to import JSON files.

We're considering allowing either a direct import `import file1 "path/to/file.json"` or using some `@embed()` mechanism. The latter seems more suitable to also allow globs at some point.

**Comment by cueckoo** [comment]
This discussion has been migrated to https://github.com/cue-lang/cue/issues/980.

For more details about CUE's migration to a new home, please see https://github.com/cue-lang/cue/issues/1078.
